arquillian-chameleon-container-model/src/test/java/org/arquillian/container/chameleon/FileUtilsTest.java

```diff
@@ -0,0 +1,32 @@
+package org.arquillian.container.chameleon;
+
+
+import org.assertj.core.api.Assertions;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.InputStream;
+
+public class FileUtilsTest {
+
+    @Rule
+    public ExpectedException thrown= ExpectedException.none();
+
+    @Test
+    public void should_not_load_configuration_file() {
+        thrown.expect(IllegalStateException.class);
+        thrown.expectMessage("Could not find built-in configuration as file nor classloader resource: containers.yaml. Something is terrible wrong with the Classpath.");
```

Comment:
If we really have to say

>  Something is terrible wrong with the Classpath.

then it should be "terribly" I believe.

But to be honest this kind of message is only confusing for the user. Can't we provide some hints, e.g. "make sure this file is in the project folder" etc.